ocs_ci/helpers/sanity_helpers.py

```diff
@@ -171,6 +171,4 @@ def __init__(self):
         Initializer for Sanity class - Init CephCluster() in order to
         set the cluster status before starting the tests
         """
-        self.pvc_objs = list()
-        self.pod_objs = list()
-        self.ceph_cluster = CephClusterExternal()
+        super(SanityExternalCluster, self).__init__()
```

Comment:
The parent class initializes `self.ceph_cluster` differently (it is a `CephCluster` not a `CephClusterExternal`). We need to consider that here.

Reply:
Correct. Thanks for pointing this out, I am closing this issue temporarily.